tests/acceptance/features/webUIResharing1/reshareUsers.feature

```diff
@@ -185,77 +196,79 @@ Feature: Resharing shared files with different permissions
 
 
   Scenario: Reshare a file and folder from shared with me page
-    Given user "Alice" has shared folder "simple-folder" with user "Brian"
-    And user "Brian" has accepted the share "simple-folder" offered by user "Alice"
-    And user "Alice" has shared file "lorem.txt" with user "Brian"
-    And user "Brian" has accepted the share "lorem.txt" offered by user "Alice"
-    And user "Brian" has logged in using the webUI
+    Given user "Brian" has created file "lorem.txt"
+    And user "Brian" has shared folder "simple-folder" with user "Alice"
```

Comment:
why switching the users around?

Reply:
because https://github.com/owncloud/web/pull/4953#discussion_r611512523